jetson/teleop/src/__main__.py

```diff
@@ -133,8 +133,9 @@ def teleop_drive_callback(self, channel, msg):
                 vel_right /= abs(vel_right)
 
         command = DriveVelCmd()

```

Comment:
Would it make more sense to invert it higher up? E.g. on line 111 in teleop:
`linear = deadzone(-input.forward_back, 0.05) * input.dampen`

Reply:
@CameronTressler

Reply:
Going to push a commit to remove the changes here. Relevant changes for teleop in #1012